indica/models/equilibrium.py

```diff
@@ -0,0 +1,289 @@
+import numpy as np
```

Comment:
Agreed that having a model of a fake equilibrium is definitely useful, but the naming of this file is a bit confusing. Even though it's in a different location I have confused this file with `indica/equilibrium.py` multiple times, perhaps `example_equilibrium.py` or something similar would be better. Not particularly important, just a suggestion.
Also docstrings for the functions are needed, although if these functions are from the fake equilibrium that I wrote then feel free to create an issue for adding the docstrings and I'll make the changes and open a pull request to merge into `Marcosertoli/186 los transform rho`.